src/app/test_executive/bootstrap_test.ml

```diff
@@ -19,7 +19,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     { default with
       block_producers=
         [{balance= "1000"; timing= Untimed}; {balance= "1000"; timing= Untimed}]
-    ; num_snark_workers= 0 }
+    ; num_snark_workers= 2 }
```

Comment:
i set this to 2 just to prove that snark workers, when they do get deployed, do successfully deploy and the test can run even if it doesn't use snarks.  i can remove this

Reply:
Let's go ahead and remove this, but good to know.